frontend/app/components/people-skills-show.js

```diff
@@ -9,23 +9,54 @@ export default Component.extend({
   init() {
     this._super(...arguments);
     this.set('peopleSkillsEditing', false);
+    this.setMemberSkillset();
   },
 
+  personChanged: observer('person.peopleSkills', function() {
```

Comment:
Das wäre wohl peopleSkillsChanged :)
Brauchst du hier `this._super(...arguments)`? Was wird da überhaupt gecalled? O_o